esmvaltool/diag_scripts/ipcc_ar5/ch12_calc_IAV_for_stippandhatch.ncl

```diff
@@ -0,0 +1,376 @@
+;
+;  Description
+;     Calculate Interannual variability from piControl runs for plotting
+;     significance with stippling and hatching, save for each model
+;     (iavmode: "each") or multi-model mean only.
+;     Can either be calculated over full time period of piControl run
+;     (no periodlength given)
+;     or first averaged annually/seasonally over periodlength and then standard
+;     deviation calculated based on averaged periods.
+;
+;  Required diag_script_info attributes (diagnostics specific)
+;     time_avg:  time period to calculate IAV, e.g. annualclim, seasonalclim
+;
+;  Optional diag_script_info attributes (diagnostic specific)
+;     iavmode: calculate multi-model mean of IAV over all models or calculate
+;              and save IAV for each individual model? (mmm, each)
+;     periodlength: length of periods to calculate IAV accross, depends on
+;                   period lengths used in
+;                   ch12_calc_map_diff_mmm_stippandhatch.ncl
+;                   if not given whole time period calculated at once
+;
+;  Caveats
+;     List possible caveats or limitations of this diagnostic
+;     Features to-be-implemented shall also be mentioned here
+;
+;  Modification history
+;     20161219-A_lore_ru: remove seasonal cycle before std if seasonal
+;     20161024-A_lore_ru: adapted to ESMValTool
+;     20130501-A_sedl_ja: written for IPCC AR5 as get_natvar.ncl.
+;
+; #############################################################################
+
+; A temporary file written by the invoking Python script
+; Passes on a number of variables from Python to NCL
+load "./interface_scripts/interface.ncl"
+load "./diag_scripts/shared/statistics.ncl"
+
+begin
+  ; ##############################################################
+  ; # Fetch general parameters, set in namelist_collins13ipcc.xml#
+  ; # passed via environment variables by python code            #
+  ; ##############################################################
+  enter_msg(diag_script, "")
+
+  ;  'diag_script' and 'variables' are fetched from the
+  ;  above 'ncl.interface' file
+  log_info("++++++++++++++++++++++++++++++++++++++++++")
+  log_info(diag_script + " (var: " + variables(0) + ")")
+  log_info("++++++++++++++++++++++++++++++++++++++++++")
+
+  ;  'datasets', 'variables' and 'field_types' are fetched from the
+  ;  above 'ncl.interface' file
+  dim_MOD = dimsizes(dataset_info@dataset)
+  dim_VAR = dimsizes(variables)
+  var0 = variables(0)
+  field_type0 = field_types(0)
+
+  ; ###########################################
+  ; # References Information                  #
+  ; ###########################################
+  write_references(diag_script, \  ; script name
+                   "A_lore_ru", \  ; authors
+                   "", \           ; contributors
+                   "D_collins13ipcc",    \  ; diag_references
+                   "",          \  ; obs_references
+                   "P_crescendo")   ; proj_references
```

Comment:
Please remove or comment out this call, since it is going to be replaced by the new provenance after #649.